src/combinatorics/configuration.lean

```diff
@@ -395,6 +395,29 @@ variables (P) {L}
 lemma point_count_eq [projective_plane P L] (l : L) : point_count P l = order P L + 1 :=
 (line_count_eq (dual P) l).trans (congr_arg (λ n, n + 1) (dual.order P L))
 
+variables (P L)
+
+lemma one_lt_order [projective_plane P L] : 1 < order P L :=
+begin
+  obtain ⟨p₁, p₂, p₃, l₁, l₂, l₃, -, -, h₂₁, h₂₂, h₂₃, h₃₁, h₃₂, h₃₃⟩ := @exists_config P L _ _,
+  classical,
+  rw [←add_lt_add_iff_right, ←point_count_eq, point_count, nat.card_eq_fintype_card],
```

Comment:
Why are you using `nat.card`? For finite projective planes, it sounds like a pretty bad choice.

Reply:
In PR #10884, Yury suggested that I use `nat.card` for `point_count` and `line_count`, which is why I also used `nat.card` for `order`. But all of the projective plane results have a `fintype` assumption, so it might sense to switch to `fintype.card`.

Reply:
I think so too

Reply:
Are you suggesting that `point_count`, `line_count`, and `order` all get switched to `fintype.card`, or just `order`?

Reply:
Probably all. Are you ever going to consider infinite projective planes? And if ever you do, you won't need `line_count` because it will definitely be infinite.

The only point of `nat.card` is to defer a few `fintype` arguments. Not worth the hassle I'd say.